src/App.js

```diff
@@ -4,7 +4,26 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+  };
+
+  searchMovie = () => {
+    const { query } = this.state;
+
+    const filtredMovies = [...moviesFromServer].filter(movie => (
```

Comment:
```suggestion
    const filtredMovies = moviesFromServer.filter(movie => (
```
'filter' is a not-mutating method. Why do you create a copy?

Reply:
Yes, you are right. My mistake, sorry